### PR TITLE
Added length calculation for insert of cache rather then default to 1…

### PR DESCRIPTION
**Pull request by pkellner**
…. changed default cache from 100 to 100MB

### CODE REVIEW

**Comment by pkellner** [comment]
@timneutkens I typically let WebStorm help but it wasn't showing any issues.  Then, I ran `npm install` and it showed some issues so I fixed those and committed back to the same branch.

**Review by timneutkens — APPROVED** [review]
💯

**Review by timneutkens — APPROVED** [review]
💯

File: examples/ssr-caching/server.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ const handle = app.getRequestHandler()
 
 // This is where we cache our rendered HTML pages
 const ssrCache = new LRUCache({
-  max: 100,
+  length: function (n, key) {
+    return n.toString().length + key.toString().length
+  },
+  max: 100 * 1000 * 1000, // 100MB cache soft limit
```
Comment (timneutkens) [inline examples/ssr-caching/server.js]: Shouldn't this be `1024` instead of `1000`, not that it really matter 😄

Comment (pkellner) [reply examples/ssr-caching/server.js]: @timneutkens  Yeah, should be. I'll fix later. for some reason I'm not seeing it in canary yet. I've had updating this example on my to-do list for  a while and just needed an accomplishment of any kind today.

**Pull request merged**